src/plugins/intel_gpu/src/graph/impls/onednn/utils.cpp

```diff
@@ -204,13 +204,13 @@ dnnl::memory::format_tag convert_data_format(cldnn::format fmt) {
 }
 
 void combine_bf_with_first_spatial_dim(cldnn::layout& l) {
-    auto pshape = l.get_shape();
+    auto pshape = l.get_partial_shape();
     ov::Shape new_shape{1, 1};
     for (size_t i = 0; i < pshape.size(); ++i) {
         if (i < 2) {
-            new_shape[0] *= pshape[i];
+            new_shape[0] *= pshape[i].is_dynamic() ? 1 : pshape[i].get_length();
         } else {
-            new_shape[1] *= pshape[i];
+            new_shape[1] *= pshape[i].is_dynamic() ? 1 : pshape[i].get_length();
```

Comment:
could you explain why `1` is used if it is unknown?

Reply:
Changed the code to "new_shape[0] *= pshape[i].get_length();"